org.eclipse.lemminx/src/test/java/org/eclipse/lemminx/extensions/xsi/XSIFormatterExperimentalTest.java

```diff
@@ -71,13 +69,43 @@ public void xsiSchemaLocationSplitOnElement() throws BadLocationException {
 				"    xsi:schemaLocation=\"http://www.springframework.org/schema/beans\r\n" + //
 				"                        http://www.springframework.org/schema/beans/spring-beans.xsd\r\n" + //
 				"                        http://www.springframework.org/schema/util\r\n" + //
-				"                        http://www.springframework.org/schema/util/spring-util.xsd\">\r\n" + //
-				"\r\n" + //
-				"</beans>";
-		assertFormat(content, expected, settings);
+				"                        http://www.springframework.org/schema/util/spring-util.xsd\"> </beans>";
+		assertFormat(content, expected, settings, te(5, 24, 6, 8, ""),

```

Comment:
for each te line, add `//` at the end of the line and format the full Java file.